Sources/Quick/QuickSelectedTestSuiteBuilder.swift

```diff
@@ -0,0 +1,69 @@
+/**
+ Responsible for building a "Selected tests" suite. This corresponds to a single
+ spec, and all its examples.
+ */
+internal class QuickSelectedTestSuiteBuilder: QuickTestSuiteBuilder {
+
+    /**
+     The test spec class to run.
+     */
+    let testCaseClass: AnyClass!
+
+    /**
+     For Objective-C classes, returns the class name. For Swift classes without,
+     an explicit Objective-C name, returns a module-namespaced class name
+     (e.g., "FooTests.FooSpec").
+     */
+    var namespacedClassName: String {
```

Comment:
Yeah, it's just that for Objective-C the class name isn't module namespaced! 😄 I suppose I was worrying whether this names is descriptive to the point of describing something that's not necessarily true 😉

What about `testCaseClassName` or `testSuiteClassName`?